vms/vm-under-test/scripts/customize-vm

```diff
@@ -22,3 +22,6 @@ set -e
 systemctl disable NetworkManager-wait-online
 systemctl disable sshd
 systemctl disable irqbalance
+
+dnf --enablerepo=rt install -y tuned-profiles-realtime
+dnf --enablerepo=nfv install -y tuned-profiles-nfv-guest
```

Comment:
AFAIU, these are not third party, but reside in additional repos.

Reply:
DONE

Reply:
The commit message still shows "third party".

Reply:
changed the phrasing